CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Added `map<k,v>` support to the Worker Inspector window. [#1403](https://github.com/spatialos/gdk-for-unity/pull/1403)
 - Added `Open inspector V2` menu item that opens the new inspector in the browser. [#1407](https://github.com/spatialos/gdk-for-unity/pull/1407)
+- A warning will be generated when enums defined in schema does not start from 0. C# enums will be generated to start from 0 and shifted to schema values on serialization and shifted back to C# values on deserialization. [#1412](https://github.com/spatialos/gdk-for-unity/pull/1412)
```

Comment:
Technically this could be a breaking change, so we should probably put it in a `### Breaking Change` section. Also maybe flip the two sentences?